app/views/pages/privacypolicy.html.erb

```diff
@@ -0,0 +1,223 @@
+<% provide(:title, 'Privacy Policy') %>
+
+<% cache do %>
+<div class="container">
+  <h1 class=about_h1>The Archives Unleashed Privacy Policy </h1>
+    <p class="about_p">We recognize the importance of and are committed to protecting the privacy of all users. The following privacy policy guides and outlines the Archives Unleashed Project’s online information practices.</p>
+    <p class="about_p">The Archives Unleashed Privacy Policy describes what users can expect from Archives Unleashed as to how information is collected, used and shared. This policy applies to all information collected from or submitted to the Archives Unleashed Project, including the Archives Unleashed Cloud (AUK) portal located at <%= link_to('cloud.archivesunleashed.org','https://cloud.archivesunleashed.org/', target: '_blank') %> and all related sub-domains.</p>
+    <p class="about_p">By accessing and using the services and products of the Archives Unleashed Project, users accept the practices outlined in the policy below.</p>
+    <p class="about_p"><strong>Information Collection and Use</strong></p>
+    <p class="about_p">We request the minimum amount of personal information necessary for the operation of Archives Unleashed software. We collect several types of information for various purposes to provide and improve our service to you.</p>
+    <div id="auk_deriv_accordionContainer accordion">
+      <div class="auk_deriv_accord card">
+        <div class="auk_deriv_accord-header card-header" id="headingOne">
+          <h5 class="mb-0">
+            <button class="btn btn-link" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">-- Personal Data</button>
```

Comment:
`-- Personal Data`
->
`Personal Data` 

(we don't need the leading `--` in the accordion headers)